test/compiler/AbstractInterpreter.jl

```diff
@@ -442,7 +442,8 @@ function custom_lookup(mi::MethodInstance, min_world::UInt, max_world::UInt)
             end
         end
     end
-    return CONST_INVOKE_INTERP.code_cache.dict[mi]
+    # XXX: This seems buggy, custom_lookup should probably construct the absint on demand.
```

Comment:
It's hacky. We need a way to set a contextual `AbstractInterpreter` globally.

Reply:
As far as I understand it AbstractInterpreters are ephemeral. Since they capture the world they operate in on construction.